src/Event.h

```diff
@@ -7,6 +7,7 @@
 
 #include "zeek/Flare.h"
 #include "zeek/IntrusivePtr.h"
+#include "zeek/RunState.h"
```

Comment:
We should avoid that RunState.h here and in EventHandler.h. It pull in all the runstate "globals" and are pretty high up headers included/used in lots of places.

Can do instead what Val.h does, declaring network_time as extern and limiting the exposure that way:
https://github.com/zeek/zeek/blob/75245bd3656fdaf7db229d8e476677bb13fdefd4/src/Val.h#L58-L64

I'm happy to fix that up during the merge, whatever you prefer.